_includes/landing-page/landing-contributors.html

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+    <div class="columns is-multiline">
+        <div class="column is-12 is-spaced-bottom has-text-centered has-text-right-desktop">
+            <h1 class="title is-size-3 is-size-2-desktop has-text-grey-darker">
+                <a class="contributors-link" href="https://github.com/openfaas" target="_blank">#TeamServerless </a>
+                - Our community and contributors of 160+
+            </h1>
+        </div>
+
+        <div id="github-users" class="user-rows">
+            <!-- Content automatically injected -->
+            <!-- {% for column in site.data.github_stats.contributorsColumns %}
```

Comment:
If we no longer need this, could you delete the dead code. It may cause confusion later. Bonus points for adding a comment pointing to where the content is generated/loaded from: `js/contributors.js` making it easier to maintain later on